contentcuration/contentcuration/frontend/shared/data/index.js

```diff
@@ -15,13 +15,15 @@ export function setupSchema() {
     console.warn('No resources defined!'); // eslint-disable-line no-console
   }
   // Version incremented to 2 to add Bookmark table and new index on CHANGES_TABLE.
-  db.version(2).stores({
+  // Version incremented to 3 to add PAGINATION_TABLE.
+  db.version(3).stores({
     // A special table for logging unsynced changes
     // Dexie.js appears to have a table for this,
     // but it seems to squash and remove changes in ways
     // that I do not currently understand, so we engage
     // in somewhat duplicative behaviour instead.
     [CHANGES_TABLE]: 'rev++,[table+key],server_rev',
+    [PAGINATION_TABLE]: 'queryString',
```

Comment:
The string for each table is a comma separated list of indexed properties, with the first listed one being the primary key. As we are caching the results of paginated queries, the queryString should uniquely identify the request in question. Although perhaps, I should have called it 'queryURL' as I think that's more what I am using, I think (if I'm not, that's a good thing to pull me up on).

Reply:
Hrm, it does rather look like I forgot to add the table or API endpoint into this, so I'll need to update that (even though we are only currently paginating ContentNode).

Reply:
Have updated this now so that the primary key is a compound key of the table and the querystring, and updated the querying and saving accordingly too.